### ISSUE TITLE
Test integration

### ISSUE THREAD

**Issue by nwolverson** [nwolverson/vscode-ide-purescript#16]
Not sure exactly what is desirable here, comments please!

**Comment by texastoland** [comment]
I think just a command mirroring the current build?

**Comment by nwolverson** [comment]
So a 2nd command that is "build and test"  ala `pulp test` or just run tests ? What should the output look like?

Simplest thing maybe to output to an output pane

**Comment by texastoland** [comment]
Yeah just output pane. It might also be nice to `--include test` in the current build command. Otherwise your stuck having to run tests (or provide a custom build command) while working on tests.

**Comment by nwolverson** [comment]
Do you know if `--include test` happily ignores missing directory? If so would seem harmless to add that to the default

**Comment by texastoland** [comment]
You're right it doesn't. I suppose I could open an issue on Pulp but it'd probably be up to me to implement.

**Comment by nwolverson** [comment]
Some hope with https://github.com/Microsoft/vscode/issues/9505

**Comment by nwolverson** [comment]
If anybody's interested in this I played around with something surfacing test results
